src/cci/cci_network.c

```diff
@@ -1352,3 +1444,50 @@ connect_srv (unsigned char *ip_addr, int port, char is_retry, SOCKET * ret_sock,
   *ret_sock = sock_fd;
   return CCI_ER_NO_ERROR;
 }
+
+static int
+createSSL (T_CON_HANDLE * con_handle, SOCKET sock_fd)
+{
+  SSL *ssl = NULL;
+  SSL_CTX *ctx = NULL;
+  int err_code = CCI_ER_NO_ERROR;
+
+  if (sock_fd == INVALID_SOCKET)
+    {
+      err_code = CCI_ER_COMMUNICATION;
+      return err_code;
```

Comment:
I think variable err_code is not required, I prefer **`return CCI_ER_COMMUNICATION;`**